### ISSUE TITLE
events - skipped heading level

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#99]
There seems to be no h2 level headings here https://rrchnm.org/category/events/. 

I only see h1 and h3 (WAVE). Is it possible to switch the h3 to an h2?
![Screen Shot 2020-10-06 at 1 27 59 PM](https://user-images.githubusercontent.com/618307/95238791-05688b80-07d8-11eb-9399-6902f7f848fd.png)
![Screen Shot 2020-10-06 at 1 28 19 PM](https://user-images.githubusercontent.com/618307/95238792-06012200-07d8-11eb-8186-d7f8a8935105.png)

**Comment by kimisgold** [comment]
This should be ready to test.

**Comment by kimisgold** [comment]
Okay, the h2s and h3s should be in the proper order now.
